src/univariate/discrete/negativebinomial3.jl

```diff
@@ -0,0 +1,118 @@
+"""
+Shape-scale parameterization of the negative binomial distribution. For small values of
+`β`, this confers superior numerical stability in comparison to the parameterization in terms
+`r`, `p` found in `NegativeBinomial`.
+
+```math
+P(X = n) = {n + \\alpha - 1 \\choose \\alpha - 1} (\\frac{1}{\\beta + 1})^\\alpha (\\frac{\\beta}{\\beta + 1})^n, \\quad \\text{for } n = 0,1,2,\\ldots
+```
+
+In terms of the mixture definition, this corresponds to:
+```math
+n \\sim \\text{Poisson}(z), \\quad
+z \\sim \\text{Gamma}(\\alpha, \\beta)
+```
+
+```julia
+NegativeBinomial3()        # distribution with α = 1.0 and β = 1.0
+NegativeBinomial3(α, β)    # distribution with shape α and scale β
```

Comment:
Indeed, BDA3's `Gamma(α, β)` is in terms of shape and inverse scale. The pmf for the above is obtained by substituting `β_dist = 1 / β` into the BDA definition. The β in the docstring refers to scale; perhaps it would be a good idea to re-name it to θ for clarity.